wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/SelectCommand.java

```diff
@@ -72,7 +72,7 @@ public SelectCommand(Supplier<Command> toRun) {
   @Override
   public void initialize() {
     if (m_selector != null) {
-      if (!m_commands.keySet().contains(m_selector.get())) {
+      if (!m_commands.containsKey(m_selector.get())) {
```

Comment:
I spot another source of bugs here: `m_selector.get()` is invoked twice. This should be rolled into a `getOrDefault` call.